occurrence-download/src/main/java/org/gbif/occurrence/download/file/simplecsv/SimpleCsvDownloadActor.java

```diff
@@ -105,7 +105,6 @@ private void doWork(DownloadFileWork work) throws IOException {
                                                        recordMap.get(DcTerm.license.simpleName()));
             //write results
             csvMapWriter.write(recordMap, COLUMNS);
-            csvMapWriter.flush();
```

Comment:
this has to be updated from the dev branch

Reply:
good catch!